subsys/bluetooth/controller/Kconfig

```diff
@@ -607,6 +615,15 @@ config BT_CTLR_ADV_PERIODIC
 	  Enable support for Bluetooth 5.0 LE Periodic Advertising in the
 	  Controller.
 
+config BT_CTLR_ADV_PERIODIC_RSP
+	bool "LE Periodic Advertising with Responses in Advertising State"
+	depends on BT_BROADCASTER && BT_CTLR_ADV_PERIODIC_RSP_SUPPORT
+	select BT_CTLR_CHAN_SEL_2
```

Comment:
missing `select EXPERIMENTAL`. same for SYNC

Reply:
What would we be marking as experimental in that case? The zephyr controller does not implement PAwR and in sdk-nrf we are planning production support in the next release.

Reply:
whoops I misread I though I was reading the host kconfig. It's all good then